src/util/ValidityUtils.cpp

```diff
@@ -2,6 +2,7 @@
 // under the Apache License, Version 2.0. See the COPYING file at the root
 // of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
 
+#include "util/ValidityUtils.h"
```

Comment:
I think it probably makes the most sense to just incorporate this into TransactionUtils.{h,cpp}. Sorry I missed responding to your earlier comment about this.